ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,271 @@
+#!/bin/bash
+char="#"
+debug='> /dev/null 2>&1'
+sys_type="yum"
+
+logger() {
```

Comment:
Maintain unity in the format of the script, the function definitions should be:
```
function(){

<commands>
```

Reply:
Done in 71fd88cf61ad859245d50e315178f0058cb61754